test/fitnesse/junit/JUnitXMLPerPageTestListenerTest.java

```diff
@@ -0,0 +1,287 @@
+package fitnesse.junit;
+
+import org.gradle.api.tasks.testing.TestDescriptor;
+import org.gradle.api.tasks.testing.TestResult;
+import org.junit.Test;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+
+public class JUnitXMLPerPageTestListenerTest {
+
+  private JUnitXMLTestResultRecorder testResultRecorderMock = mock(JUnitXMLTestResultRecorder.class);
+
+  // SUT
+  private JUnitXMLPerPageTestListener jUnitXMLPerPageTestListener = new JUnitXMLPerPageTestListener(testResultRecorderMock);
+
+  @Test(expected = Test.None.class /* no exception expected */)
```

Comment:
Why do you need to make explicit that no exception is thrown. Isn't that the default?

Reply:
Fixed. Please revise.
@fhoeben what do you prefer: Should I resolve the conversations when I have fixed the issues or would you like to do this after checking my changes?

Reply:
You can just mark as resolved.